changes/21777-list-fleet-manated-apps

```diff
@@ -0,0 +1 @@
+- Add API endpoint to list team available fleet managed apps
```

Comment:
nit: since this is customer facing, probably a good idea to stick with "Fleet-maintained apps" for consistency?

Reply:
```suggestion
- Add API endpoint to list team available Fleet-maintained apps
```